src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.cpp

```diff
@@ -309,18 +364,23 @@ bool FlightTaskAutoFollowTarget::update()
 	}
 
 	// Publish status message for debugging
-	_target_position_filter.getState().copyTo(follow_target_status.pos_est_filtered);
 	follow_target_status.timestamp = hrt_absolute_time();
+	_target_pose_filter.getState().copyTo(follow_target_status.pos_est_filtered);
+	_target_pose_filter.getRate().copyTo(follow_target_status.vel_est_filtered);
+
+	follow_target_status.tracked_target_orientation = _target_orientation_rad;
+	follow_target_status.follow_angle = _follow_angle_rad;
 	follow_target_status.emergency_ascent = _emergency_ascent;
 	follow_target_status.gimbal_pitch = _gimbal_pitch;
+
 	_follow_target_status_pub.publish(follow_target_status);
 
 	_constraints.want_takeoff = _checkTakeoff();
 
 	return true;
 }
 
-float FlightTaskAutoFollowTarget::update_follow_me_angle_setting(int param_nav_ft_fs) const
+float FlightTaskAutoFollowTarget::get_follow_me_angle_setting(int param_nav_ft_fs) const
```

Comment:
can we make this more precise and demand the specific enum type? e.g. `const FollowPerspective &follow_perspective`